packages/pn-commons/src/components/AppResponseMessage.tsx

```diff
@@ -12,13 +12,19 @@ import { AppResponsePublisher } from '../utility/AppResponse';
  * Make sure that you have the necessary Redux setup and reducers in place for this code to work correctly.
  * @returns {any}
  */
-const AppResponseMessage = () => {
+type Props = {
+  eventTrackingToastErrorMessages?: (error: AppResponseError, traceid?: string) => void;
+};
+const AppResponseMessage = ({ eventTrackingToastErrorMessages }: Props) => {
   const dispatch = useDispatch();
 
   const showErrorMessage = (response: AppResponse) => {
-    const { errors, action } = response;
+    const { errors, action, traceId } = response;
 
     errors?.forEach((error) => {
+      if (eventTrackingToastErrorMessages) {
+        eventTrackingToastErrorMessages(error, traceId);
```

Comment:
@NicolaGiornetta manca la property "page_name = nome pagina in cui è apparso il toast di errore"

Reply:
la page_name è definita direttamente nella funzione di tracking con i parametri. Tra l'altro con le CR sarà rifattorizzata.